src/core/webview/ClineProvider.ts

```diff
@@ -1508,7 +1514,7 @@ export class ClineProvider
 		}
 	}
 
-	async getStateToPostToWebview() {
+	async getStateToPostToWebview(): Promise<ExtensionState> {
```

Comment:
I was surprised to see that we didn't have very strict type safety on `getState` and `getStateToPostToWebview`.